README.md

```diff
@@ -1504,7 +1504,21 @@ var event = await query.executeQuery();
 console.log(`>> Event: ${JSON.stringify(event)}`);
 ```
 
-
+# File Upload
+This API is used to upload a file and return the metadata that needs to be saved in delivery. Currently, this API is supported only in Browsers. 
+```
+await client.fileUploader.upload(file); 
+```
+where file is an instance of [File](https://developer.mozilla.org/en-US/docs/Web/API/File), will return
+```
+{
+  name: <name-referenceed-by-file-object>, // file.name
```

Comment:
Check for typos: referenceed => referenced
What is the purpose of comments such as "file.name"? They do not seem to bring much value